Formula/agda.rb

```diff
@@ -109,17 +78,17 @@ def caveats
 
     if build.with? "stdlib"
       s += <<-EOS.undent
-      To use the Agda standard library, point Agda to the following include dir:
-        #{HOMEBREW_PREFIX}/lib/agda/src
-      EOS
+      The Agda standard library has been installed to:
+        #{HOMEBREW_PREFIX}/lib/agda
 
-      if build.with? "malonzo"
-        s += <<-EOS.undent
+      Agda 2.5.1 includes new library management features.
+      http://agda.readthedocs.org/en/stable/tools/package-system.html
```

Comment:
This explains why we’re suggesting the user to create never-before used files in their home directory, without which their existing setup _will_ break.